doc/content/source/userobjects/ThresholdHeuristicUserObject.md

```diff
@@ -0,0 +1,24 @@
+# ThresholdHeuristicUserObject
+
+## Description
+
+`ThresholdHeuristicUserObject` inherits from the `ClusteringHeuristicUserObjectBase`. It implements the clustering
```

Comment:
I don't think you need to describe the class hierarchy in the documentation - this information is already well covered in the Doxygen documentation.